src/generator/AutoRest.Go/MethodScopeProvider.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace AutoRest.Go
 {
+    /// Refactor -> Namer
     public class MethodScopeProvider
```

Comment:
This sort of thing shouldn't be necessary anymore, disambiguation is done on the fly. We should probably review why this is still here and find a way to remove it.